python/pfs/drp/stella/reduceArcTask.py

```diff
@@ -226,5 +226,4 @@ def run(self, expRefList, butler, wLenFile=None, lineList=None, immediate=True):
             pfsArm = spectrumSetToPfsArm(pfsConfig, spectrumSetFromProfile,
                                          dataId["visit"], dataId["spectrograph"], dataId["arm"])
             butler.put(PfsArmIO(pfsArm), 'pfsArm', dataId)
-
```

Comment:
Please don't include pure-white-space changes.  

Is the commit comment correct?  Both changes look like whitespace.